src/main.js

```diff
@@ -0,0 +1,25 @@
+function iterator(array) {
```

Comment:
2 things here:

1. `iterator` is not a descriptive function name. What about something like `convertItemsToUnorderedList`?
2. `array` is not a descriptive argument name and also might cause confusion with the [`Array` object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array). What about something like `items`?